op-service/txmgr/queue_test.go

```diff
@@ -209,15 +225,30 @@ func TestQueue_Send(t *testing.T) {
 					TxData: []byte{byte(i)},
 					To:     &common.Address{},
 				}
+				if i == 0 {
+					// Make the first tx much larger to expose
+					// any race conditions in the queue
+					for j := 0; j < 100_000; j++ {
+						candidate.TxData = append(candidate.TxData, 1)
+					}
```

Comment:
This looks quite inefficient. Maybe we just do something like
```suggestion
					candidate.TxData = make([]byte, 100_000)
```
? Zeros should be good enough for this purpose.